app/scripts/modules/amazon/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -418,5 +425,46 @@ module.exports = angular.module('spinnaker.serverGroup.details.aws.controller',
         serverGroup.accountDetails = details;
       });
     };
+
+    this.lookForPipelineCommitDiffs = (executionId) => {
```

Comment:
It would be nice to move this logic into the `viewChangesLink` component instead of pushing more stuff into this controller, which is already > 400 LOC.